### PR TITLE
completed solution16

### PR DESCRIPTION
**Pull request by Jayshgandhi**
Output:

Scalar multiplication:	[936]
Vector multiplication:	[115 402 801 225 ]
Matrix multiplication:
[	[6	12	1	]
	[20	42	4	]
	[1	4	9	]
]

### CODE REVIEW

File: Exercises/Excercise16/solution16.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import java.util.Arrays;
+
+class Multiply{
+    public Integer process(Integer n, Integer y){
+        return n*y;
+    }
+    public Integer[] process(Integer[]n, Integer[] y){
+        Integer k[] = new Integer[n.length];
+        try {
+            if (n.length == y.length) {
```
Comment (fr-judson) [inline Exercises/Excercise16/solution16.java:11]: "If" case is not passed exception should be thrown.

File: Exercises/Excercise16/solution16.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        }
+        catch(Exception e){
+            System.out.println("error");
+        }
+        return k;
+    }
+    public Integer[][] process(Integer[][]n, Integer[][]y){
+        Integer p[][] = new Integer[n.length][y.length];
+        try {
+            if (n.length == y.length) {
```
Comment (fr-judson) [inline Exercises/Excercise16/solution16.java:30]: "if" case is not passed exception should be thrown.

**Review by fr-judson — CHANGES_REQUESTED** [review]
Logic is perfect and You should raise exception manually using throw keyword in the else block.
Understand about throw keyword in Java fix the issue.

**Review by hockeyjudson — APPROVED** [review]
LGTM

**Pull request merged**